app/views/migration/_form.html.erb

```diff
@@ -1,5 +1,6 @@
 <% with_options scope: 'site.campaigns.migration.form' do |i18n| %>
-  <%= simple_form_for @aggregation, url: { action: 'create' }, builder: Migration::FormBuilder do |agg| %>
+  <% extra_form_html = { 'recaptcha-site-key': Recaptcha&.configuration&.site_key! } unless current_user %>
```

Comment:
This will be needed elsewhere in future, so let's put it into a helper module instead.